apps/BasicExample/android/gradle.properties

```diff
@@ -10,7 +10,7 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx512m -XX:MaxMetaspaceSize=256m
-org.gradle.jvmargs=-Xmx4096m -XX:MaxMetaspaceSize=512m
+org.gradle.jvmargs=-Xmx2048m -XX:MaxMetaspaceSize=512m
```

Comment:
TBH, I don't remember why FabricExample had more memory given to it. We can try reducing it.

Reply:
Seems that 512 is enough 😅 

Done in [81737e9](https://github.com/software-mansion/react-native-gesture-handler/pull/3490/commits/81737e92962e78d514821af80239f88eabd21e18)